src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -234,13 +243,34 @@ class ExternalIssueActionList extends AsyncComponent {
             <Modal.Header closeButton>
               <Modal.Title>{`${selectedIntegration.provider.name} Issue`}</Modal.Title>
             </Modal.Header>
+            <ul
+              className="nav nav-tabs"
+              style={{borderBottom: '1px solid rgb(221, 221, 221)'}}
+            >
+              <li className={action === 'create' ? 'active' : ''}>
+                <a onClick={() => this.handleClick('create')}>{t('Create')}</a>
+              </li>
+              <li className={action === 'link' ? 'active' : ''}>
+                <a onClick={() => this.handleClick('link')}>{t('Link')}</a>
+              </li>
+            </ul>
             <Modal.Body>
-              <ExternalIssueForm
-                group={this.props.group}
-                integration={selectedIntegration}
-                action={action}
-                onSubmitSuccess={this.closeModal}
-              />
+              {action === 'create' && (
+                <ExternalIssueForm
+                  group={this.props.group}
+                  integration={selectedIntegration}
+                  action={'create'}
```

Comment:
instead of having 2 ternaries, can't we just render `ExternalIssueForm` with `action={action}`?